src/capi/atom.cpp

```diff
@@ -182,7 +182,7 @@ extern "C" chfl_status chfl_atom_set_property(CHFL_ATOM* const atom, const char*
     )
 }
 
-extern "C" CHFL_PROPERTY* chfl_atom_get_property(const CHFL_ATOM* const atom, const char* name) {
+extern "C" CHFL_PROPERTY* chfl_atom_get_property(const CHFL_ATOM* atom, const char* name) {
```

Comment:
But the const is useful (-ish) here: the compiler will check that we don't reset the pointer to another location.